client/src/utilities/api-calls.js

```diff
@@ -26,4 +26,21 @@ const addNewEntry = (newEntry) => {
   .catch(error => console.log(error));
 }
 
-export { getAllEntries, addNewEntry };
\ No newline at end of file
+const updateEntry = (updatedEntry, id) => {
+  return fetch(`http://localhost:3001/api/v1/entries/${id}`, {
+    method: 'PUT',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({journal_entry: updatedEntry})
```

Comment:
looks like POST and PUT methods are super similar! great job at implementing this